src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,66 @@
+@import '../utils/veriables';
+
+header {
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+  box-shadow: 0 2px 4px #0000000d;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-right: 419px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+li {
+  margin-right: 20px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  color: #060b35;
+  font-size: 12px;
+  line-height: 60px;
+  text-decoration: none;
+  display: block;
+  transition: transform $tramsform-time;
```

Comment:
There is a typo in the variable name `$tramsform-time`. It should likely be `$transform-time`. Ensure the variable is correctly defined in the imported file.